ucm2/DB845c/HiFi.conf

```diff
@@ -0,0 +1,46 @@
+# Use case configuration for DB845c.
+# Author: Srinivas Kandagatla <srinivas.kandagatla@linaro.org>
+
+SectionVerb {
+
+	EnableSequence [
+		cdev "hw:${CardId}"
+		cset "name='SLIMBUS_0_RX Audio Mixer MultiMedia2' 1"
+
+		<codecs/wcd934x/DefaultEnableSeq.conf>
+		<codecs/wsa881x/DefaultEnableSeq.conf>
+	]
+
+	DisableSequence [
+                cdev "hw:${CardId}"
+		<codecs/wcd934x/DefaultDisableSeq.conf>
+		cset "name='SLIMBUS_0_RX Audio Mixer MultiMedia2' 0"
+	]
+
+	Value {
+		TQ "HiFi"
+	}
+}
+
+SectionDevice."Speaker" {
+	Comment "Speaker playback"
+
+	EnableSequence [
+		cdev "hw:${CardId}"
+		<codecs/wcd934x/SpeakerEnableSeq.conf>
+		<codecs/wsa881x/SpeakerEnableSeq.conf>
+	]
+
+	DisableSequence [
+		cdev "hw:${CardId}"
+		<codecs/wsa881x/SpeakerDisableSeq.conf>
+		<codecs/wcd934x/SpeakerDisableSeq.conf>
+	]
+
+	Value {
+		PlaybackPriority 100
+		PlaybackPCM "hw:${CardId},1"
+                PlaybackVolume "RX1 Digital Volume"
+                PlaybackVolume "RX2 Digital Volume"
```

Comment:
This does not work. You cannot set two values to one key. Only the last is used.